message_ix/tests/test_macro.py

```diff
@@ -208,10 +211,23 @@ def test_calibrate_roundtrip(westeros_solved):
     # this is a regression test with values observed on Aug 9, 2019
     with_macro = westeros_solved.add_macro(W_DATA_PATH, check_convergence=True)
     aeei = with_macro.par("aeei")["value"].values
-    npt.assert_allclose(aeei, 1e-3 * np.array([20, 71.759022, 37.424904, 19.936694]))
+    npt.assert_allclose(
+        aeei,
+        1e-3
+        * np.array([20, -7.514287920171298, 43.62562814596503, 21.143463116337838]),
```

Comment:
The default relative tolerance for [`assert_allclose`](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_allclose.html) is 1e-7, so we don't need to include more than that many significant figures.